ofmeet/src/java/org/jivesoftware/openfire/plugin/ofmeet/ConfigServlet.java

```diff
@@ -202,6 +232,107 @@ public void doGet( HttpServletRequest request, HttpServletResponse response ) th
         }
     }
 
+	private void handleAuthenticatedUser(JSONObject config, HttpServletRequest request, String xmppDomain, JSONObject conferences)
+	{
+		String userName = request.getUserPrincipal().getName();
+
+		config.put( "id", userName + "@" + xmppDomain );
+
+		try {
+			User user = XMPPServer.getInstance().getUserManager().getUser(userName);
+
+			config.put( "emailAddress", user.getEmail() );
+			config.put( "nickName", user.getName() );
```

Comment:
Perhaps we should further improve on this, and see if a nickname has been set in the VCard, in a similar way as how you get the avatar. I believe that's how Spark saves the nickname. 

Later, we could add functionality that queries the more modern nickname and avatar storage.